lib/watir/extensions/alerts.rb

```diff
@@ -5,7 +5,7 @@ module Watir
   #
   # Module provided by optional require:
   #
-  #   require "watir-webdriver/extensions/alerts"
+  #   require "watir/extensions/alerts"
```

Comment:
Well, this is the conversion not necessarily the new release. I'd actually like to incorporate your locator changes as part of Watir 6.0 release to differentiate it from the final watir-webdriver release (and especially if we make non-backward compatible changes). I agree this would be a good time to remove old alert implementation.